src/OpenLoco/Paint/PaintStation.cpp

```diff
@@ -0,0 +1,168 @@
+#pragma once
+#include "PaintStation.h"
+#include "../CompanyManager.h"
+#include "../Graphics/Colour.h"
+#include "../Interop/Interop.hpp"
+#include "../Map/Tile.h"
+#include "../Objects/ObjectManager.h"
+#include "../Objects/TrainStationObject.h"
+#include "../Station.h"
+#include "../Ui.h"
+#include "Paint.h"
+
+namespace OpenLoco::Paint
+{
+    using paintStationTrackFunc = void (*)(PaintSession& session, const Map::StationElement& elStation);
+
+    void nullPaintFunc(PaintSession& session, const Map::StationElement& elStation) {}
+    std::array<std::array<paintStationTrackFunc, 44>, 1> trainStationFuncs = { {
+        paintFunc4D79AC,
+        paintFunc4D7A5C,
```

Comment:
This is for diagonals which you can't actually place so looks to be dead code.

Reply:
Turns out some people hack their objects to allow this so will need to implement.